pkgs/applications/audio/bitwig-studio/bitwig-studio1.nix

```diff
@@ -67,22 +76,11 @@ stdenv.mkDerivation rec {
       -not -name '*.so' \
       -not -path '*/resources/*' | \
     while IFS= read -r f ; do
-      patchelf \
-        --set-interpreter $(cat ${stdenv.cc}/nix-support/dynamic-linker) \
-        $f && \
       wrapProgram $f \
         --prefix PATH : "${binPath}" \
-        --prefix LD_LIBRARY_PATH : "${libPath}" \
+        "''${gappsWrapperArgs[@]}" \
         --set LD_PRELOAD "${libxkbcommon.out}/lib/libxkbcommon.so" || true
```

Comment:
It might be a bit cleaner though to just extend `gappsWrapperArgs` and remove `dontWrapGApps`

```
gappsWrapperArgs+=("--prefix" "PATH" ":" "${binPath}")
gappsWrapperArgs+=("--set" "LD_PRELOAD" "${libxkbcommon.out}/lib/libxkbcommon.so")
```